validation.js

```diff
@@ -4,6 +4,30 @@ const message = document.createElement('span');
 message.classList.add('error-message');
 message.textContent = '*Please, the email address must be in lowercase';
 errorMessage.appendChild(message);
+const userName = document.getElementById('username');
+const descriptionTextarea = document.getElementById('msg-input');
+const validate = document.querySelectorAll('.input');
+const formData = {
+  userName: JSON.parse(localStorage.getItem('formData')).userName,
+  userEmail: JSON.parse(localStorage.getItem('formData')).userEmail,
+  message: JSON.parse(localStorage.getItem('formData')).message,
+};
+
+const renderInfo = () => {
+  const storedUserData = JSON.parse(localStorage.getItem('formData'));
+  userName.value = storedUserData.userName;
+  emailInput.value = storedUserData.userEmail;
+  descriptionTextarea.value = storedUserData.message;
+};
+const localStore = (event, value) => {
+  formData[value] = event.target.value;
+  localStorage.setItem('formData', JSON.stringify(formData));
+};
```

Comment:
- [ ] I appreciate your effort, but the form data is not getting persisted to the localStorage. Kindly check the function you are using to save the data to localStorage. ✍️

<a href="https://www.loom.com/share/b7a8ce8ca41f4a07bae407a6a514467c">
    <p>Luis Sanchez - 4 February 2023 - Watch Video</p>
    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/b7a8ce8ca41f4a07bae407a6a514467c-with-play.gif">
  </a>